config/task.go

```diff
@@ -286,11 +290,13 @@ func (c *TaskConfig) Validate() error {
 		switch cond := c.Condition.(type) {
 		case *ServicesConditionConfig:
 			if cond.Regexp != nil && *cond.Regexp != "" {
-				log.Printf("[ERR] (config.task) list of services and service condition regex " +
-					"both provided. If both are needed, consider including the list in the regex " +
-					"or creating separate tasks.")
-				return fmt.Errorf("task.services is not allowed if task.condition.regexp " +
+				err := fmt.Errorf("task.services is not allowed if task.condition.regexp " +
 					"is configured for a services condition")
+				logging.Global().Named(logSystemName+taskSubsystemName).Error("[ERR] (config.task) list of "+
+					"services and service condition regex "+
+					"both provided. If both are needed, consider including the list in the regex "+
+					"or creating separate tasks.", "error", err)
```

Comment:
```suggestion
				logging.Global().Named(logSystemName+taskSubsystemName).Error("[ERR] (config.task) list of "+
					"services and service condition regex both provided. If both are needed, consider "+
					"including the list in the regex or creating separate tasks.", "error", err)
```

a nice-to-have to reduce lines where we can but still keep it readable.